### PR TITLE
Updating /grow, disabling comment warnings in commands.cs.

### PR DESCRIPTION
**Pull request by csmir**
Naming tile types & comment unavailable types in /grow, practical disabling of XML warnings in the commands class.

### CODE REVIEW

File: TShockAPI/Commands.cs [hunk 1]
```diff
@@ -6301,108 +6304,111 @@ public static void Grow(CommandArgs args)
...
+					WorldGen.TryGrowingTreeByType(TileID.TreeAmber, x, y);
 					name = "Amber Gemtree";
 					break;
 
-				case "cactus":
-					Main.tile[x, y].type = 53;
-					WorldGen.GrowCactus(x, y);
-					name = "Cactus";
-					break;
+                // Commenting out these two cases until a fix has been found.
```
Comment (hakusaro) [inline TShockAPI/Commands.cs:6387]: Can you just remove them if they're broken? We can always restore them with git. If you make two separate commits it'll be easy.

File: TShockAPI/TShockAPI.csproj [hunk 2]
```diff
@@ -229,7 +229,7 @@
   </PropertyGroup>
   <ProjectExtensions>
     <VisualStudio>
-      <UserProperties BuildVersion_IncrementBeforeBuild="False" BuildVersion_StartDate="2011/6/17" BuildVersion_BuildVersioningStyle="None.None.None.MonthAndDayStamp" BuildVersion_BuildAction="Both" BuildVersion_UpdateFileVersion="True" BuildVersion_UpdateAssemblyVersion="True" />
+      <UserProperties BuildVersion_UpdateAssemblyVersion="True" BuildVersion_UpdateFileVersion="True" BuildVersion_BuildAction="Both" BuildVersion_BuildVersioningStyle="None.None.None.MonthAndDayStamp" BuildVersion_StartDate="2011/6/17" BuildVersion_IncrementBeforeBuild="False" />
```
Comment (hakusaro) [inline TShockAPI/TShockAPI.csproj:232]: Can you avoid committing changes to the `csproj` unintentionally?

File: CHANGELOG.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ This is the rolling changelog for TShock for Terraria. Use past tense when addin
   * If there is no section called "Upcoming changes" below this line, please add one with `## Upcoming changes` as the first line, and then a bulleted item directly after with the first change.
 
 ## Upcoming changes
+* Type naming for tile & wall ID's in `/grow`, Removing cases: 'cactus' & 'herb' until a fix is found. (@Rozen4334)
```
Comment (hakusaro) [inline CHANGELOG.md:16]: Can you add these to the newest upcoming changes section at the bottom?

**Review by hakusaro — CHANGES_REQUESTED** [review]
This looks pretty good, but it also looks like it conflicts with another pending PR? Can you take a look at #2400? In fact, I'd be interested to see what @drunderscore thinks about this...? It looks to me like #2400 might be [the wrong abstraction](https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction)?

**Comment by drunderscore** [comment]
@hakusaro I noticed only after I submitted my PR of this ones existence, but we talked about it in Discord [here](https://canary.discord.com/channels/479657350043664384/711282444933791774/865476773856411648) and he was okay with me simply incorporating the changes into my PR to supersede this one.

Whether or not my PR is following the wrong abstraction, I do actually see the concern there and the problematic nature of it. I will stand by my PR, mostly because I've already written the code, but by all means if it isn't the proper way to make the code more concise we can leave as-is.

**Comment by hakusaro** [comment]
> @hakusaro I noticed only after I submitted my PR of this ones existence, but we talked about it in Discord here and he was okay with me simply incorporating the changes into my PR to supersede this one.

This is good to know. In your PR, do you mind adding him as part of your changelog item and/or removing the broken things if they're still broken?
